terraform/README.md

```diff
@@ -25,7 +25,7 @@ like shown below:
 
 ```text
 module "ausf" {
-  source = "git::https://github.com/canonical/sdcore-ausf-k8s-operator/terraform/modules/terraform-juju-sdcore-ausf-k8s"
+  source = "git::https://github.com/canonical/sdcore-ausf-k8s-operator/terraform"
```

Comment:
https://developer.hashicorp.com/terraform/language/modules/sources#modules-in-package-sub-directories

As the module is located in a repo subdirectory `//terraform` needs to be used according to guide. This is also tested and works in this way.

Reply:
This is only applicable when the module is in a subdirectory of a package. It's not the case here.